### PR TITLE
Feature/280 implement sbt client

### PR DESCRIPTION
**Pull request by ITStar10**

### CODE REVIEW

File: packages/vda-sbt-client/src/blockchain/blockchainApi.ts [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+export interface SBTClientConfig {
+    identifier: string;
+    signKey: string;
+    chainNameOrId?: string | number;
+
+    callType: CallType;
+    web3Options: VeridaSelfTransactionConfig | VeridaMetaTransactionConfig;
+}
+
+export interface SBTInfo {
```
Comment (tahpot) [inline packages/vda-sbt-client/src/blockchain/blockchainApi.ts:29]: @ITStar10 Add documentation that explains the purpose of each attribute

File: packages/vda-sbt-client/test/blockchain-api.test.ts [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+const createBlockchainAPI = (did: any) => {
+    return new BlockchainApi({
+        identifier: did.address,
+        signKey: did.privateKey,
+        chainNameOrId: "testnet",
+        ...configuration
+    })
+}
+
+describe('vda-sbt-client blockchain api', () => {
```
Comment (tahpot) [inline packages/vda-sbt-client/test/blockchain-api.test.ts:22]: I'm getting timeouts for all of these. Can you set timeout to be 20 seconds for each test?

I tried setting `this.timeout()`, however it won't work ... which is weird as it works on all our other test suites.

**Review by tahpot — CHANGES_REQUESTED** [review]
See comments

File: packages/vda-sbt-client/test/blockchain-api.test.ts (hunk 2 above)
Comment (tahpot) [reply packages/vda-sbt-client/test/blockchain-api.test.ts:22]: Is working locally now.

File: packages/vda-sbt-client/src/blockchain/blockchainApi.ts (hunk 1 above)
Comment (ITStar10) [reply packages/vda-sbt-client/src/blockchain/blockchainApi.ts:29]: Updated.

**Review by tahpot — APPROVED** [review]

**Pull request merged**